django_restql/mixins.py

```diff
@@ -354,7 +355,7 @@ def get_related_fields(related_fields_mapping, dict_parsed_restql_query):
         related_fields = []
         for key, related_field in related_fields_mapping.items():
             fields = key.split(".")
-            if isinstance(related_field, str):
+            if isinstance(related_field, (str, Prefetch)):
```

Comment:
Please add tests for this.
Documentation should also be updated, this section https://yezyilomo.github.io/django-restql/#syntax-for-prefetch_related-and-select_related